sources/tools/Stride.Importer.3D/Material/Materials.cs

```diff
@@ -0,0 +1,158 @@
+                if (assimp.GetMaterialIntegerArray(material, MatKeyTexTypeBase, (uint)type, (uint)iEl, ref elType, ref pMax) != Return.Success)
+                    elType = (int)StackElementType.Texture;//continue; // error !
+
+                switch ((StackElementType)elType)
+                {
+                    case StackElementType.Operation:
+                        if (assimp.GetMaterialIntegerArray(material, Silk.NET.Assimp.Assimp.MaterialTexopBase, (uint)type, (uint)iEl, ref elOp, ref pMax) != Return.Success)
+                        {
+                            logger?.Error("Material not found");
+                            continue; // error !
+                        }
+                        el = new StackOperation(ConvertAssimpStackOperationCppToCs[elOp], elAlpha, elBlend, elFlags);
+                        break;
+                    case StackElementType.Color:
+                        if (assimp.GetMaterialColor(material, MatKeyTexColorBase, (uint)type, (uint)iEl, ref elColor) != Return.Success)
+                        {
+                            logger?.Error("Material with index not found");
+                            continue; // error !
+                        }
+                        el = new StackColor(new Color3(elColor.X, elColor.Y, elColor.Z), elAlpha, elBlend, elFlags);
+                        break;
+                    case StackElementType.Texture:
+                        if (assimp.GetMaterialString(material, Silk.NET.Assimp.Assimp.MaterialTextureBase, (uint)type, (uint)iEl, ref elTexPath) != Return.Success)
+                            continue; // error !
+                        if (assimp.GetMaterialIntegerArray(material, Silk.NET.Assimp.Assimp.MaterialUvwsrcBase, (uint)type, (uint)iEl, ref elTexChannel, ref pMax) != Return.Success)
+                            elTexChannel = 0; // default channel
+                        if (assimp.GetMaterialIntegerArray(material, Silk.NET.Assimp.Assimp.MaterialMappingmodeUBase, (uint)type, (uint)iEl, ref elMappingModeU, ref pMax) != Return.Success)
+                            elMappingModeU = (int)TextureMapMode.Wrap; // default mapping mode
+                        if (assimp.GetMaterialIntegerArray(material, Silk.NET.Assimp.Assimp.MaterialMappingmodeVBase, (uint)type, (uint)iEl, ref elMappingModeV, ref pMax) != Return.Success)
+                            elMappingModeV = (int)TextureMapMode.Wrap; // default mapping mode
+
+                        el = new StackTexture(
+                            elTexPath.AsString,
+                            elTexChannel,
+                            ConvertAssimpMappingModeCppToCs[elMappingModeU],
+                            ConvertAssimpMappingModeCppToCs[elMappingModeV],
+                            elAlpha,
+                            elBlend,
+                            elFlags);
+                        break;
+                    default:
+                        // error !
```

Comment:
Log the error here as well, something like `$"{(StackElementType)elType} not implemented."`